### PR TITLE
send team IDs when creating a new collection

### PR DESCRIPTION
**Pull request by gedge**
### What

send team IDs when
- creating a new collection
- and new auth is enabled

### How to review

Describe the steps required to test the changes.

### Who can review

Describe who worked on the changes, so that other people can review.

### CODE REVIEW

File: src/app/views/collections/create/CreateNewCollection.jsx [hunk 1]
```diff
@@ -146,6 +146,17 @@ const CreateNewCollection = props => {
...
+    const makeTeams = () => {
+        if (isNewSignIn) {
+            return newCollection.teams.map(team => {
+                return team.id;
+            });
+        }
+        return newCollection.teams.map(team => {
+            return team.name;
+        });
+    };
```
Comment (rafahop) [inline src/app/views/collections/create/CreateNewCollection.jsx:158]: Would the following be simpler?
```suggestion
        return newCollection.teams.map(team => {
            if (isNewSignIn) {
                return team.id;
            } else {
                return team.name;
            }
        });
    };
```

Comment (rafahop) [reply src/app/views/collections/create/CreateNewCollection.jsx:158]: Also, would it be possible to add unit test coverage for the new sign in?

Comment (gedge) [reply src/app/views/collections/create/CreateNewCollection.jsx]: I've added some unit tests for my change

File: src/app/views/collections/create/CreateNewCollection.jsx [hunk 2]
```diff
@@ -146,6 +146,15 @@ const CreateNewCollection = props => {
         return new Date(`${newCollection.publishDate.value} ${newCollection.publishTime.value}`).toISOString();
     };
 
+    const makeTeams = () => {
+        return newCollection.teams.map(team => {
```
Comment (joegasewicz) [inline src/app/views/collections/create/CreateNewCollection.jsx:150]: Maybe a comment explaining what has changed in the passed in props or what is expected by the api with the PUT

**Review by joegasewicz — APPROVED** [review]
Justneed to rebase develop

**Review by joegasewicz — APPROVED** [review]

**Pull request merged**